docs/gauntlet/withdraw.md

```diff
@@ -0,0 +1,17 @@
+# Withdraw Payments using Gauntlet
+
+When Node Operators report data to a feed, they are compensated with LINK thats owned by the feed. Node operators can then withdraw the funds owed to them via gauntlet using this section below
+
+## Sample Command Usage
+
+Make sure to set up gauntlet using the steps described in [Gauntlet Setup](gauntlet-setup.md) before attempting to run the following command
+
+```bash
+yarn gauntlet ocr2:withdraw_payment --network=mainnet --recipient=<VALID_LINK_RECIPIENT_ADDRESS> <FEED_ADDRESS>
```

Comment:
What does `VALID_LINK_RECIPIENT_ADDRESS` mean? I think it means that it is the associated token account...is that correct? 

Would be good to also add something like `VALID_LINK_RECIPIENT_ADDRESS` = the same account address used in setting the payee config (should be the associated account for the LINK token specific to the owner public key)

Reply:
Update it @aalu1418, lmk if it's more clear now